File: src/main/java/com/example/demo/model/BlacklistIp.java
package com.example.demo.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BlacklistIp {
    private Long id;
    private String ipAddress;
    private OffsetDateTime createdAt;
    private Boolean deleted;
    private Integer times;
}

File: src/main/kotlin/com/example/demo/model/BlacklistIp.kt
package com.example.demo.model

import java.time.OffsetDateTime

data class BlacklistIp(
    val id: Long? = null,
    val ipAddress: String,
    val createdAt: OffsetDateTime? = null,
    val deleted: Boolean? = null,
    val times: Int? = null
)
